### PR TITLE
XWIKI-14215: fix some missing blog translation

### PR DESCRIPTION
**Pull request by Pbast**

### CODE REVIEW

File: application-blog-ui/src/main/resources/Blog/CategorySheet.xml [hunk 1]
```diff
@@ -47,7 +47,7 @@
 #set ($obj = $doc.getObject($blogCategoryClassname))
 #if ($obj)
   #getEntriesForCategory($doc.fullName $discard $totalEntries)
-  = Category: $services.rendering.escape($obj.getValue('name'), $doc.syntax) ($totalEntries posts) [[#toolImage('feed')&gt;&gt;Blog.CategoryRss||queryString="xpage=plain&amp;category=$escapetool.url($doc.fullName)" title="RSS"]] =
+  = $services.localization.render('xe.blog.sheet.category') $services.rendering.escape($obj.getValue('name'), $doc.syntax) ($totalEntries posts) [[#toolImage('feed')&gt;&gt;Blog.CategoryRss||queryString="xpage=plain&amp;category=$escapetool.url($doc.fullName)" title="RSS"]] =
```
Comment (evalica) [inline application-blog-ui/src/main/resources/Blog/CategorySheet.xml]: The 'xe' prefix is no longer valid. Make sure the translation exist in https://github.com/xwiki-contrib/application-blog/blob/master/application-blog-ui/src/main/resources/Blog/Translations.xml or create one.

Comment (tmortagne) [inline application-blog-ui/src/main/resources/Blog/CategorySheet.xml]: Should be 'blog.sheet.category'.

**Comment by tmortagne** [comment]
It has nothing to do with XWiki version. The translations are in the application itself, see the link indicated by @evalica.

**Comment by Pbast** [comment]
You said "It has nothing to do with XWiki version." but Blog.Translations on xwiki 8.4.4 (and blog Application 8.4.4) contain only 4 translation keys but https://github.com/xwiki-contrib/application-blog/blob/master/application-blog-ui/src/main/resources/Blog/Translations.xml (branch master) containing more than 100 translation keys...
I don't know if we could install blog 9.x on xwiki 8.4.4.
My patch did not working on xwiki 8.4.4 with Blog Application 8.4.4 but will working well on next blog version... I suppose (because master branch Translations.xml containing correct translation key and checked on http://l10n.xwiki.org too)
Then , is my patch is ok now?

**Comment by tmortagne** [comment]
This pull request is on blog application contrib extension, not platform. And the translation comes with the application here whatever is the version of XWiki on which it's installed (and according to the pom.xml it's compatible with XWiki 8.4.4).

**Pull request merged**